### DISCUSSION TITLE
Automation of issue flow between projects

### DISCUSSION THREAD

**Discussion by AgnieszkaUcinska** [community/community#135981] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I have the following use case:
I have a team working in different repos. We have one shared Project Board, where all the issues from all repos are visible (Project A). However we have a need for a separate board for triaging bugs for one of the repos. I have set up a Project (Project B) to cover that repo only. It has a different issue flow than the main board. The different statuses are updated nicely in the Issues (as they should be). 
And now for the challenge:
I would like to connect and automate the flows between the boards so we do not have to move issues around manually. For example when we move a ticket to Backlog on the triage board (Project B) it will move to Ready on the main board by itself (Project A). When we move it to In Progress it moves to In Development and so on. 
The out of the box automations are of no use here. Any ideas if this is doable without too much of a hassle?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by GabrielAlmeidag** [comment, marked as answer]
Hi @AgnieszkaUcinska 👋

To automate moving issues between different project boards on GitHub, you have a few options. One effective method is to use GitHub Actions. You can set up a custom GitHub Action that triggers when an issue is moved to a specific column in Project B (your triage board) and then automatically moves it to the corresponding column in Project A (your main board). This approach requires some coding but provides a high level of control.

Alternatively, you could explore using third-party tools like Zapier or Automate.io. These tools can connect GitHub with other services to create more complex workflows, potentially allowing you to automate the movement of issues between boards without manual intervention.

Another option is to use GitHub Webhooks combined with a custom server setup. Webhooks can monitor changes in Project B and trigger updates in Project A through the GitHub API. While these solutions might require some setup, they can help streamline the workflow between your project boards.
